subs/pantry/test/Pantry/HPackSpec.hs

```diff
@@ -0,0 +1,20 @@
+module Pantry.HPackSpec (spec) where
+
+import Test.Hspec
+import Pantry
+import Pantry.HPack
+import Path
+
+isVersion :: Version -> Bool
+isVersion _ = True
+
+spec :: Spec
+spec = describe "Parse HPack version" $ do
+  it "Shipped hpack version" $ do
+    version <- runPantryApp $ hpackVersion
+    version `shouldSatisfy` isVersion
+
+  -- it "External hpack version" $ do
+  --   fp <- parseAbsFile "/home/sibi/.local/bin/hpack"
+  --   version <- runPantryAppWithCustomHpack fp hpackVersion
+  --   version `shouldSatisfy` isVersion
```

Comment:
do we need to have it as a comment?

Reply:
If we don't have it as a comment, it will fail in CI. :-) Basically, this tests needs an external hpack executable for it to work. We need to modify the CI scripts for it to actually work. So, right now I have added the basic idea of the test. We can adapt it when we actually integrate it with the CI.